apps/examples/testcase/tc_main.c

```diff
@@ -25,6 +25,7 @@
 #include <sched.h>
 #include <sys/types.h>
 #include <semaphore.h>
+#include <apps/shell/tash.h>
 
 #if defined(CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_UTC) || defined(CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_ITC)
 #define TC_ARASTORAGE_STACK       4096
```

Comment:
When CONFIG_TASH is enabled, these definitions are not needed. Let's check a conditional.